TTS/config/shared_configs.py

```diff
@@ -210,6 +210,13 @@ class BaseTrainingConfig(Coqpit):
         tb_model_param_stats (bool):
             Enable / Disable logging internal model stats for model diagnostic. It might be useful for model debugging.
             Defaults to ```False```.
+            wandb_disabled: bool = False
+        wandb_project_name (str):
+            Name of the W&B project. Defaults to config.model
+        wandb_entity (str):
+            Name of W&B entity/team. Enables collaboration across a team or org.
+        wandb_log_model_step (int):
```

Comment:
How is this different than `save_step` ?

Reply:
`save_step` is meant for local checkpointing. This is meant to log checkpoints in cloud( you W&B project dashboard. See how it [looks ](https://wandb.ai/cayush/Tacotron2/artifacts/model/rvjlrp7t_checkpoint/da6007b538e933eab163)). We can use `save_step` for both things but I thought it'd be better to let the user decide as logging models take bandwidth. 
We can go with whatever you prefer. Let me know :)

Reply:
I think it is good to keep it but maybe set it to `save_step`if not defined.

Reply:
Okay

Reply:
done